lib/vagrant/util/install_cli_autocomplete.rb

```diff
@@ -0,0 +1,82 @@
+module Vagrant
+  module Util
+    # Generic installation of content to shell config file
+    class InstallShellConfig
+      PREPEND = "".freeze
+      STRING_INSERT = "".freeze
+      APPEND = "".freeze
+      CONFIG_PATHS = [""].freeze
+
+      def self.shell_installed(home)
```

Comment:
These should all have method docs for the parameters expected and the return type expected